chsdi/models/vector/__init__.py

```diff
@@ -185,23 +185,13 @@ def get_queryable_attributes_keys(cls, lang):
             settings = get_current_registry().settings
             availableLangs = settings['available_languages'].replace(' ', '|')
             for attr in cls.__queryable_attributes__:
-                match = re.search(r'(_(%s))$' % availableLangs, attr)
-                if match is not None:
-                    # Lang specific attr
-                    suffix = '_%s' % lang
-                    suffixAttr = match.groups()[0]
-                    fallbackMatch = getFallbackLangMatch(
-                        cls.__queryable_attributes__,
-                        suffix,
-                        attr,
-                        suffixAttr
-                    )
-                    if fallbackMatch not in queryableAttributes:
-                        queryableAttributes.append(fallbackMatch)
-                else:
-                    # Not based on lang
-                    queryableAttributes.append(attr)
-        return queryableAttributes
+                fallbackMath = getFallbackLangMatch(
+                    cls.__queryable_attributes__,
+                    lang,
+                    availableLangs,
+                    attr
+                )
+        return queryableAttributes.append(fallbackMath)
```

Comment:
You need to append the `fallbackMatch` within the loop. Here you're only appending the last match.